services/api-server/tests/unit/conftest.py

```diff
@@ -101,7 +101,7 @@ def auth(mocker, app: FastAPI, faker: Faker) -> HTTPBasicAuth:
     # mock engine if db was not init
     if app.state.settings.API_SERVER_POSTGRES is None:
 
-        engine = mocker.Mock()
+        engine = mocker.MagicMock()
```

Comment:
Q: what is the difference?

Reply:
one is magic, the other don't ;-)

e.g. in this case, with Magic he auto-generate mocks for functions e.g. ``async with engine.acquire()`` would return a new ``MagicMock``. This does not happen with only Mock